include/dxc/dxcapi.h

```diff
@@ -257,8 +257,8 @@ struct IDxcLibrary : public IUnknown {
     _In_ IDxcBlob *pBlob, _COM_Outptr_ IStream **ppStream) = 0;
   virtual HRESULT STDMETHODCALLTYPE GetBlobAsUtf8(
     _In_ IDxcBlob *pBlob, _COM_Outptr_ IDxcBlobEncoding **pBlobEncoding) = 0;
-  virtual HRESULT STDMETHODCALLTYPE GetBlobAsUtf16(
```

Comment:
Isn't this still a breaking change from the original because we no longer have the `GetBlobAsUtf16` method?

Reply:
It is a breaking change, but intentional.  The code gives (and has always given) you utf16 on win32, but utf32 on *nix.  This is the reason for the rename.  Code that updates to a new header and uses the old method must be changed to use the renamed method.  This is why there's a comment indicating that the method has simply been renamed.  It's an intentional breaking change.